src/components/form-grid/form-grid.stories.js

```diff
@@ -0,0 +1,148 @@
+import { html } from 'lit-html';
+import '../form';
+import '../form-grid';
+import '../input-field';
+import '../button';
+import '../form-message';
+import styles from './styles.scss';
+import formStyling from '../form/styles.scss';
+import inputFieldStyling from '../input-field/styles.scss';
+import buttonStyling from '../button/styles.scss';
+import formMessageStyle from '../form-message/styles.scss';
+import { stylesheet, docsIntro } from '../../../.storybook/utils.js';
+import { args, argTypes } from '../grid/config';
+import { CATEGORIES } from '../../../.storybook/utils.js';
+
+export default {
+  title: 'native-elements/vl-form-grid',
+  decorators: [
+    (story) =>
+      html` ${stylesheet(`${styles}${formStyling}${inputFieldStyling}${buttonStyling}${formMessageStyle}`)}${story()} `,
+  ],
+  parameters: {
+    docs: {
+      description: {
+        component: docsIntro({
+          stylesheets: ['form-grid'],
+          root: 'form-grid',
+          intro: 'Class that enables a grid layout in a form.',
+        }),
+      },
+    },
+  },
+  args: {
+    ...args,
+    columnSize: 4,
+    columnsAmount: 7,
+    label: 'Dit is een label',
+  },
+  argTypes: {
+    ...argTypes,
+    columnsAmount: {
+      name: 'amount of columns (for demo purposes)',
+      control: { type: 'range', min: 1, max: 12, step: 1 },
+      table: {
+        type: { summary: 'string' },
+        category: CATEGORIES.ATTRIBUTES,
+        defaultValue: { summary: '' },
+      },
+    },
+    columnSize: {
+      name: 'size of the columns (for demo purposes)',
+      table: {
+        type: { summary: 'string' },
+        category: CATEGORIES.ATTRIBUTES,
+        defaultValue: { summary: '' },
+      },
+    },
+    label: {
+      name: 'label of the input field (for demo purposes)',
+      table: {
+        type: { summary: 'string' },
+        category: CATEGORIES.ATTRIBUTES,
+        defaultValue: { summary: '' },
+      },
+    },
+  },
+};
+
+export const Default = ({
```

Comment:
Deze story mag voor mij een pak simpeler. Een demo zoals dit lijkt me voldoende om te laten zien wat de component doet.

https://webcomponenten.omgeving.vlaanderen.be/demo/vl-form-grid.html

Ook DV doet het zo: https://overheid.vlaanderen.be/webuniversum/v3/documentation/form-grid

Het aantal columns aanpassen en het label aanpassen en zo verder lijkt me ook minder van toepassing hier. Het klopt ook niet helemaal want als ik het aantal columns op 1 zet zie ik er nog 2. Ik de story ook niet in een `vl-section` en `vl-layout` enzo steken, want dat heeft hier weinig mee te maken. Bij de `vl-grid` was dat anders, daar lieten we echt de opbouw van een pagina zien. By the way, als je `for demo purposes` props gebruikt (wat volgens mij geen meerwaarde heeft in deze story) zorg er dan voor dat die niet onder de categorie `attributes` vallen want dat is niet zo.